manifests/fragment.pp

```diff
@@ -23,20 +23,24 @@
 #   Deprecated
 # [*backup*]
 #   Deprecated
+# [*show_diff*]
+#   Boolean whether to show fragment-diffs or not.
 #
 define concat::fragment(
     $target,
-    $content = undef,
-    $source  = undef,
-    $order   = '10',
-    $ensure  = undef,
-    $mode    = undef,
-    $owner   = undef,
-    $group   = undef,
-    $backup  = undef
+    $content   = undef,
+    $source    = undef,
+    $order     = '10',
+    $ensure    = undef,
+    $mode      = undef,
+    $owner     = undef,
+    $group     = undef,
+    $backup    = undef,
+    $show_diff = true
```

Comment:
why is this true by default?
shouldn't we just pass undef down the line to `file`, so it uses `file`'s default?